roles/client_side_tests/tasks/test_e2e.yml

```diff
@@ -1,126 +1,133 @@
----
-- name: "Show the prom_url"
-  ansible.builtin.debug:
-    var: prom_url
+    - name: Get the default-prometheus-htpasswd secret
```

Comment:
The entire file has been indented. What is the reason for this?

Reply:
Don't know how it happened. Fixed.